Readme.md

```diff
@@ -91,12 +93,39 @@ $ ./y.sh test --release
 
 ## Usage
 
+**Note:** You can ignore this step, by just cd into the repo directory.
+
 `$CG_GCCJIT_DIR` is the directory you cloned this repo into in the following instructions:
 
 ```bash
 export CG_GCCJIT_DIR=[the full path to rustc_codegen_gcc]
 ```
 
+Then you have to run these commands, in the corresponding order:
+
+```bash
+1- ./y.sh prepare
+2- ./y.sh build
+```
+If you get an error while running the second command, you should delete the first line inside of your ``config.toml`` like the following:
```

Comment:
Please remove this part. I'll update the `config.example.toml` to comment the `gcc-path` setting instead.

Reply:
I did not delete the prepare and build commands, since that something that we have to do first to run this command: `` ./y.sh cargo build --manifest-path tests/hello-world/Cargo.toml ``

Reply:
Perhaps we should rework the previous part of the readme as those commands are shown there.

@GuillaumeGomez: How would you see this?

Reply:
I think it's fine for now, we can always make changes about it later on.